### ISSUE TITLE
"Invalid" waystones

### ISSUE THREAD

**Issue by noobanidus** [duely/w2w#3]

**Comment by NielsPilgaard** [comment]
I've received multiple reports of waypoints named invalid being created by this mod, here's the related bug report: https://github.com/NillerMedDild/Enigmatica5/issues/338

**Comment by noobanidus** [comment]
Yeah. I just haven't had the time/energy to sit down and figure this out. At this point in time I'd recommend not including the mod with your pack, as I don't know when I'll have time to work on it.

**Comment by NielsPilgaard** [comment]
Alright, thank you for being honest, I will remove it for now :)
